modules/kubernetes/config/Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM java:8
 ENV IGNITE_VERSION 2.0.0-SNAPSHOT
 
 # Set IGNITE_HOME variable.
-ENV IGNITE_HOME /opt/ignite/apache-ignite-lgpl-${IGNITE_VERSION}-bin
```

Comment:
let's change it to /opt/gridgiain/gridgain-community-${IGNITE_VERSION}, also below

Reply:
Agreed. Done.